templates/master/00-master/on-prem/files/keepalived-script-both.yaml

```diff
@@ -3,4 +3,4 @@ path: "/etc/kubernetes/static-pod-resources/keepalived/scripts/chk_ocp_script_bo
 contents:
   inline: |
     #!/bin/bash
-    /usr/bin/curl -o /dev/null -kLfs https://localhost:{{`{{ .LBConfig.LbPort }}`}}/readyz && [ -e /var/run/keepalived/iptables-rule-exists ] || /usr/bin/curl -kLfs https://localhost:{{`{{ .LBConfig.ApiPort }}`}}/readyz
+    /usr/bin/curl -o /dev/null --max-time 1.9 -kLfs https://localhost:{{`{{ .LBConfig.LbPort }}`}}/readyz && [ -e /var/run/keepalived/iptables-rule-exists ] || /usr/bin/curl --max-time 1.9 -kLfs https://localhost:{{`{{ .LBConfig.ApiPort }}`}}/readyz
```

Comment:
There are actually 2 checks, so we should have `--max-time` < 1 to ensure this fits in the 2 second interval at which the tests are run.

Reply:
wait they are run concurrently (&&), i think that 1.9 seconds for each should be ok

Reply:
jk, I always thought that && let you run each task as a thread, but aparently its a conditional chain of operations :zany_face: